### PR TITLE
put test_message to 8.0 version

### PR DESCRIPTION
**Pull request by albertsh**
- some corrections done in the module's code:
 - write can take only one argument
 - self.env.with_context doesn't work
 - add of paranthesis

### CODE REVIEW

File: contract_compassion/tests/test_base_module.py [hunk 1]
```diff
@@ -21,10 +21,10 @@ class test_base_module(test_base_contract):
     def setUp(self):
         # Retrieve of income account
         super(test_base_module, self).setUp()
-        property_account_income = self.env['account.account'].search([
+        self.property_account_income = self.env['account.account'].search([
```
Comment (ecino) [inline contract_compassion/tests/test_base_module.py]: Why changing this ? I see no use...

Comment (albertsh) [inline contract_compassion/tests/test_base_module.py]: because we use it in a method latter (_create_gift)

Comment (ecino) [inline contract_compassion/tests/test_base_module.py]: Ok,

**Comment by ecino** [comment]
:+1:

**Pull request merged**